devices.js

```diff
@@ -14054,6 +14054,35 @@ const devices = [
     },
 
     // Legrand
+    {
+        zigbeeModel: [' Contactor\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'+
+            '\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000',
+        ],
+        model: 'FC80CC',
+        description: 'Legrand (or Bticino) DIN contactor module (note: Legrand 412171 may be similar to Bticino FC80CC)',
+        vendor: 'Legrand',
+        extend: generic.switch,
+        fromZigbee: [
+            fz.identify, fz.on_off, fz.electrical_measurement_power, fz.legrand_device_mode, fz.ignore_basic_report, fz.ignore_genOta,
+        ],
+        toZigbee: [
+            tz.legrand_deviceMode, tz.on_off, tz.legrand_identify, tz.legrand_readActivePower,
+        ],
+        exposes: [
+            e.switch().withDescription('On/Off (works only if device is in "switch" mode)'),
+            exposes.enum( 'device_mode', exposes.access.ALL, ['switch', 'auto'])
```

Comment:
why is dimmer_on/off not in this list?

Reply:
Because this device support only switch & auto mode
I have anticipated fromZigbee & toZigbee code for other "Legrand" devices, and also to avoid multiplicity of code
if you prefer I simply remove dimmer_on and dimmer_off ?

Reply:
Ah I see now, no you can leave it. Please check my other comments.

Reply:
ok, and yes will apply your other comments